__tests__/__snapshots__/index.ts.snap

```diff
@@ -540,7 +540,7 @@ Array [
   },
   Object {
     "durationMs": 117,
-    "durationMsPretty": "4ms",
+    "durationMsPretty": "<placeholder pretty ms duration>",
```

Comment:
This was a mistake from earlier – this value should always have been constantized into a placeholder. But it took me until now for the timing to be different enough for me to notice. :smile: